hm_1.py

```diff
@@ -0,0 +1,24 @@
+"""
+Calculate sum of all digits of a number set in string.
+
+Example:
+
+num = �12345�
+
+Your script should calculate it like: 1+2+3+4+5 = 15
+
+num = �611091234512�
+result = 6+1+1+0+9+1+2+3+4+6+1+2 = 36
+"""
+def sum_str(n):
+    sum_line = 0
+    for i in n:
+        sum_line += int(i)
+    print("���� ������ {n} = {sum_line}".format(n=n,sum_line=sum_line) )
```

Comment:
I suggest using `return` instead of `print`.

Also, what are the strange symbols here?

Reply:
Probably encoding problem. Corrected